CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [`USC`](https://github.com/software-mansion/universal-sierra-compiler) - before it used to consume CASM artifacts
 produced by Scarb if they were present. Setting up `casm = true` in `Scarb.toml` is no longer recommended - it may slow
 down the compilation.
+- The `replace_bytecode` cheatcode now returns `Result` with a possible `ReplaceBytecodeError`
```

Comment:
I would describe why it can return an error now

Reply:
I mean, it's not the docs

Reply:
Sure, but it is confusing why an error can appear here if we didn't report any before

Reply:
I did some explaining but idk if that what's you meant